common/server.conf

```diff
@@ -8,3 +8,17 @@ ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:
 ssl_prefer_server_ciphers on;
 
 server_name dcos.*;
+
+location /system/logs/v1/ {
+    access_by_lua 'auth.validate_jwt_or_exit()';
+    proxy_buffering off;
+    proxy_cache off;
+
+    proxy_set_header Connection '';
+    proxy_http_version 1.1;
+    chunked_transfer_encoding off;
+
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $host;
```

Comment:
Could you please also add here `proxy_set_header X-Forwarded-Proto $scheme` ? It will most probably not be used by the upstream, but it would be nice if we started thinking about a common set of headers we always pass to upstreams.